sample/db/samples/taxons.rb

```diff
@@ -2,6 +2,10 @@
 
 Spree::Sample.load_sample('taxonomies')
 
+ADDITIONAL_TAXONS = ['Bestsellers', 'Trending', 'Streetstyle', 'Summer Sale']
+
+SPECIAL_TAXONS = { 'New Collection': 'Summer 2021', 'Special Offers': '30% Off' }
+
```

Comment:
@aplegatt it's up to you, but maybe it's worth to `freeze` array and hash here, even if we only iterate over them